src/components/WwwFrame/TheTipMessage.vue

```diff
@@ -21,37 +58,94 @@ export default {
 		query: tipMessageData,
 		preFetch: true,
 		result({ data }) {
-			if (this.$refs.tip) {
-				const showing = data?.tip?.visible ?? false;
-
-				if (showing) {
-					const message = data?.tip?.message ?? '';
-					this.safeMessage = DOMPurify.sanitize(message, { ALLOWED_TAGS: ['b', 'i', 'em', 'strong', 'a'] });
-					this.messageType = data?.tip?.type ?? '';
-					this.persist = data?.tip?.persist ?? false;
-
-					this.$refs.tip.show(this.safeMessage, this.messageType, this.preventAutoDismiss);
-					this.initialPath = this.$route.path;
-				} else {
-					this.$refs.tip.close();
-				}
-			}
+			const showing = data?.tip?.visible ?? false;
+			this.tipMessage = showing ? data?.tip ?? null : null;
 		}
 	},
 	data() {
 		return {
-			safeMessage: '',
-			messageType: '',
-			persist: false,
-			initialPath: null
+			initialPath: null,
+			flashMessages: [],
+			tipMessage: null,
 		};
 	},
+	created() {
+		this.initialPath = this.$route.path;
+
+		// query for flash messages
+		this.apollo.query({
+			query: flashMessageQuery,
+			variables: {
+				visitorId: this.cookieStore.get('uiv'),
+			},
+		}).then(({ data }) => {
+			const tips = data?.tips ?? [];
+			const warnings = data?.warnings ?? [];
+			const errors = data?.errors ?? [];
+			this.flashMessages = [...errors, ...warnings, ...tips];
+		}).then(() => {
+			// clear all flash messages
+			return this.apollo.mutate({
+				mutation: clearFlashMessagesMutation,
+				variables: {
+					visitorId: this.cookieStore.get('uiv'),
+				},
+			});
+		}).catch(err => {
+			try {
+				console.error(err);
```

Comment:
`logFormatter`?

Reply:
I was going to say that this is only for the browser console, in which case I prefer to just spit out the error using the formatting provided by `console.error`, but I realized this is actually in created so it could get called on the server. I'm going to move this into mounted so that it's only called in the browser.